libs/features/mas/web-components/build.mjs

```diff
@@ -0,0 +1,107 @@
+import { writeFileSync } from 'node:fs';
+import { build } from 'esbuild';
+import { execSync } from 'node:child_process';
+
+// Get the current commit hash
+const commitHash = execSync('git rev-parse HEAD').toString().trim();
+const branch = execSync('git rev-parse --abbrev-ref HEAD').toString().trim();
+console.log(`you're building from branch ${branch} with commit ${commitHash}`);
+const params = process.argv.slice(2);
+const outfolder = params.includes('milo') ? '../../../../libs/deps/' : '../libs';
```

Comment:
i don't think we should make the separation here anymore

Reply:
also i wonder again if this shouldn't be the occasion to move mas deps to libs/deps/mas, leaving libs/deps/* the time consumer update their dependencies

Reply:
Ah, I forgot to ask this today. Thanks for confirming, it's now removed.